File: src/main/java/me/laus/tallinn_transport/util/FileSystemUtils.java
package me.laus.tallinn_transport.util;

import java.nio.file.Paths;

public class FileSystemUtils {
    private final static String CURRENT_PATH = Paths.get("").toAbsolutePath().toString();

    public final static String DEFAULT_STOPS_PATH = CURRENT_PATH + "/src/main/resources/stops.txt";
}

File: src/main/java/me/laus/tallinn/transport/util/FileSystemUtils.kt
package me.laus.tallinn.transport.util

import java.nio.file.Paths

object FileSystemUtils {
    private val CURRENT_PATH = Paths.get("").toAbsolutePath().toString()
    @kotlin.jvm.JvmField
    val DEFAULT_STOPS_PATH = CURRENT_PATH + "/src/main/resources/stops.txt"
}